src/client/scripts/room.js

```diff
@@ -1,41 +1,44 @@
 "use strict";
 /* global Cards:false Rooms:false*/
 /**
-*@author THouse
 *@purpose To provide the room template with data to display
 **/
 
 Template.room.onCreated(function () {
-    this.subscribe("cards");
+    Meteor.autorun(function() {
```

Comment:
I don't think this is necessary, 
auto-subscribe is happening in CardMethods.js line 10